spec/acceptance/lastEvaluatedKeyWithFilterAndPagingTest.spec.ts

```diff
@@ -143,15 +159,16 @@ describe('lastEvaluatedKeyWithFilterAndPagingTest', function ()
         assert.isDefined(page2.firstEvaluatedKey);
         assert.isUndefined(page2.lastEvaluatedKey);
 
-        let page1Again = await nodenamo.list().from(User).filter(filter).limit(2).resume(page2.firstEvaluatedKey).execute<User>();
+        let page1Again = await nodenamo.list().from(User).filter(filter).limit(2).order(false).resume(page2.firstEvaluatedKey).execute<User>();
 
         assert.deepEqual(page1Again.items, [
-            user1,
-            user3
+            user3,
+            user1
         ]);
-        assert.isDefined(page1Again.lastEvaluatedKey)
+        assert.isDefined(page1Again.firstEvaluatedKey)
+        assert.isUndefined(page1Again.lastEvaluatedKey)
 
-        let page2Again = await nodenamo.list().from(User).filter(filter).limit(2).resume(page1Again.lastEvaluatedKey).execute<User>();
+        let page2Again = await nodenamo.list().from(User).filter(filter).limit(2).resume(page1Again.firstEvaluatedKey).execute<User>();
```

Comment:
Could you add a similar test to https://github.com/mappies/nodenamo/blob/a3a2b0ed9a73e4d8aa664d5bc49460215fc667d1/spec/acceptance/idTest.spec.ts#L108 that would reverse the paging from 4th page back to 0th page?  That is a basic test where as this is an advance test case.

Reply:
Have you add a chance to add tests to that ?